es6-template-literals/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = 'Irvine Center Drive';
+    const address = `LearningFuze: ${number} ${street}`;
+
+    console.log(address);
```

Comment:
Please make sure you are A) doing one-to-one conversions of concatenations to template literals, and B) not modifying other parts of the code.

---

![image](https://user-images.githubusercontent.com/16143771/127083120-7a9771ea-f386-4ef8-8c46-91cea0fda25b.png)

---

![image](https://user-images.githubusercontent.com/16143771/127083147-bfb02bdb-f769-4c34-82c4-f7b8f89454a1.png)